Makefile

```diff
@@ -14,6 +14,7 @@ OUTPUT_PATH = output
 RPI_NETWORK_TYPE ?= wlan0
 RPI_HOSTNAME     ?= rpi-kube-master-01
 RPI_IP           ?= 192.168.1.101
+RPI_GATEWAY      ?= 192.168.1.1
 RPI_DNS          ?= 192.168.1.1
```

Comment:
What do you think about maybe setting this to the `RPI_GATEWAY` variable as default unless otherwise set?

Reply:
That could work too. Actually, I think setting `RPI_DNS = $(RPI_GATEWAY)` would be less risky in terms of safer defaults.

When my Pi tried to use my DNS server as its gateway, I definitely couldn't reach it externally.
If my Pi didn't have DNS, but at least a functioning gateway config, I could at least access it remotely but discover named resolution would fail (causing me to look at /etc/resolv.conf or something). And at that point, I would go back to the `makefile` and set a value for `RPI_DNS`.

Reply:
Yeah thats exactly what I meant. Sorry about the misunderstanding. Perfect. Lets get this in :)